cmd/minikube/cmd/start.go

```diff
@@ -822,21 +821,21 @@ func validateMemorySize(req int, drvName string) {
 			out.V{"requested": req, "mininum": minUsableMem})
 	}
 	if req < minRecommendedMem && !viper.GetBool(force) {
-		out.T(out.Notice, "Requested memory allocation ({{.requested}}MB) is less than the recommended minimum {{.recommended}}MB. Kubernetes may crash unexpectedly.",
+		out.WarningT("Requested memory allocation ({{.requested}}MB) is less than the recommended minimum {{.recommended}}MB. Kubernetes may crash unexpectedly.",
 			out.V{"requested": req, "recommended": minRecommendedMem})
 	}
 
 	if driver.IsDockerDesktop(drvName) {
 		// in Docker Desktop if you allocate 2 GB the docker info shows:  Total Memory: 1.945GiB which becomes 1991 when we calculate the MBs
 		// thats why it is not same number as other drivers which is 2 GB
 		if containerLimit < 1991 {
-			out.T(out.Tip, `Increase Docker for Desktop memory to at least 2.5GB or more:
+			out.WarningT(`Increase Docker for Desktop memory to at least 2.5GB or more:
 			
 	Docker for Desktop > Settings > Resources > Memory
 
 `)
 		} else if containerLimit < 2997 && sysLimit > 8000 { // for users with more than 8 GB advice 3 GB
-			out.T(out.Tip, `Your system has {{.system_limit}}MB memory but Docker has only {{.container_limit}}MB. For a better performance increase to at least 3GB.
+			out.WarningT(`Your system has {{.system_limit}}MB memory but Docker has only {{.container_limit}}MB. For a better performance increase to at least 3GB.
```

Comment:
why would this be a warning? we are only suggesting the suer with a lot of ram to maybe increase their default docker desktop to 3 GB (not required but suggestion)

Reply:
So`out.T` is meant to represent a specific step in the minikube process (e.g. Starting VM, Preparing kubernetes), and that's how JSON interprets calls to `out.T`.

This log is informational or warning in nature. I could change it to `out.Infof` if you think that's more appropriate

Reply:
In general, it's probably worth it to have a doc around what the different functions in the `out` package should be used for